src/components/InfoPanel/InfoPanel.test.js

```diff
@@ -31,6 +31,22 @@ describe("Info Panel", () => {
     expect(wrapper.find("Topology").length).toBe(1);
   });
 
+  it("renders the expanded topology on click", () => {
+    const store = mockStore(dataDump);
+    const wrapper = mount(
+      <Provider store={store}>
+        <MemoryRouter initialEntries={["/models/group-test"]}>
+          <TestRoute path="/models/*">
+            <InfoPanel />
+          </TestRoute>
+        </MemoryRouter>
+      </Provider>
+    );
+    expect(wrapper.find("[data-test='topology-modal']").length).toBe(0);
+    wrapper.find("i.p-icon--expand").simulate("click");
```

Comment:
Can you reference the `.p-icon--expand` inside [data-test='topology-modal']. If example:
```js
wrapper.find(".info-panel__pictogram .p-icon--expand").simulate("click");
```

There maybe more icons in the final view.

Reply:
done